docs/developers/getting-started.md

```diff
@@ -42,19 +42,23 @@ timesketch-dev         | User dev created/updated
 timesketch-dev         | Timesketch development server is ready!
 ```
 
-Add a user to your Timesketch server (this will add a user `dev` with password `dev`)
+Per default a user `dev` with password `dev` is created for you. If you want to add additional users to your Timesketch server, run the following command:
 
 ```bash
-$ docker-compose exec timesketch tsctl create-user dev --password dev
-User dev created/updated
+$ docker-compose exec timesketch tsctl create-user <USER> --password <PW>
+User <USER> created/updated
 ```
 
-Now, start the `gunicon` server that will serve the Timsketch WSGI app
+### Web server
+
+Now, start the `gunicon` server that will serve the Timsketch WSGI app.
+
+To make this task easier, we recommend using the `timesketch/contrib/tsdev.sh` script.
 
 In one shell:
 
 ```bash
-$ docker-compose exec timesketch gunicorn --reload -b 0.0.0.0:5000 --log-file - --timeout 120 timesketch.wsgi:application
```

Comment:
how about we keep both? I know tsdev is maybe easier, but having the raw command here seems still useful to understand what happens.

Reply:
Yeah, why not :)